control_msgs/action/Gripper.action

```diff
@@ -0,0 +1,6 @@
+bool open
```

Comment:
What if you want to control the percentage of the closure?

Reply:
That's a good idea. I will update it.